mlflow/tracking/fluent.py

```diff
@@ -39,7 +39,32 @@ def set_experiment(experiment_name):
     Set given experiment as active experiment. If experiment does not exist, create an experiment
     with provided name.
 
-    :param experiment_name: Name of experiment to be activated.
+    :param experiment_name: Case sensitive name of an experiment to be activated.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Set an experiment name, which must be unique and case sensitive.
+        mlflow.set_experiment("Social NLP Experiments")
+
+        # Get Experiment Details
+        experiment = mlflow.get_experiment_by_name("Social NLP Experiments")
+
+        # Print the contents of Experiment data
+        print("Experiment_id: {}".format(experiment.experiment_id))
+        print("Artifact Location: {}".format(experiment.artifact_location))
+        print("Tags: {}".format(experiment.tags))
+        print("Lifecycle_stage: {}".format(experiment.lifecycle_stage))
+
+    .. code-block:: text
+        :caption: Output
+
+        Experiment_id: 1
+        Artifact Location: file:///.../apis/mlruns/1
```

Comment:
```suggestion
        Artifact Location: file:///.../mlruns/1
```

Reply:
Same thing here when I run the code:

```
INFO: 'Social NLP Experiments' does not exist. Creating a new experiment
Experiment_id: 1
Artifact Location: file:///Users/julesdamji/examples/py/mlflow/apis/fluent/mlruns/1
Tags: {}
Lifecycle_stage: active
```
Better to just show ```file:///.../mlruns/1```